2024/04/21 02:17:49 [error] 386#386: *449 FastCGI sent in stderr: "PHP message: {"reqId":"LXbXCGrxjXe71HWIVf6N","level":3,"time":"2024-04-21T02:17:49+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 03:17:50 [error] 386#386: *451 FastCGI sent in stderr: "PHP message: {"reqId":"wGG2wYyzculdVyiGszF5","level":3,"time":"2024-04-21T03:17:50+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 03:37:34 [error] 386#386: *453 FastCGI sent in stderr: "PHP message: {"reqId":"VYm980ndQ1XH4z3ShKV6","level":3,"time":"2024-04-21T03:37:34+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 04:18:55 [error] 386#386: *455 FastCGI sent in stderr: "PHP message: {"reqId":"L9DNR68GmP23J4cSA5G3","level":3,"time":"2024-04-21T04:18:55+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 04:36:52 [error] 386#386: *457 FastCGI sent in stderr: "PHP message: {"reqId":"Uo5nYPDRzJIo7H8YsFOh","level":3,"time":"2024-04-21T04:36:52+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 05:09:23 [error] 386#386: *459 FastCGI sent in stderr: "PHP message: {"reqId":"eMZ5XV5ppjGcBvgnLlVf","level":3,"time":"2024-04-21T05:09:23+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 05:32:59 [error] 386#386: *461 FastCGI sent in stderr: "PHP message: {"reqId":"9cuuwfBKdcuhNZJvR8RP","level":3,"time":"2024-04-21T05:32:59+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 05:58:29 [error] 387#387: *463 FastCGI sent in stderr: "PHP message: {"reqId":"X3FSMD4mrmbvjuEsGwu5","level":3,"time":"2024-04-21T05:58:29+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 06:17:52 [error] 387#387: *465 FastCGI sent in stderr: "PHP message: {"reqId":"kpRNsU29OSGRBeuBosKR","level":3,"time":"2024-04-21T06:17:52+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 06:55:48 [error] 387#387: *467 FastCGI sent in stderr: "PHP message: {"reqId":"leKPENwmppiRa9llqz5l","level":3,"time":"2024-04-21T06:55:48+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 07:17:55 [error] 387#387: *469 FastCGI sent in stderr: "PHP message: {"reqId":"tiWh5qtLObewQbtxLayp","level":3,"time":"2024-04-21T07:17:55+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 07:49:39 [error] 387#387: *471 FastCGI sent in stderr: "PHP message: {"reqId":"XeVKKtH0L63T9OqdNTNS","level":3,"time":"2024-04-21T07:49:39+00:00","remoteAddr":"104.166.80.239","user":"--","app":"index","method":"GET","url":"/","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (X11; Linux i686; rv:109.0) Gecko/20100101 Firefox/120.0","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\DBAL\\...; PHP message: {"reqId":"XeVKKtH0L63T9OqdNTNS","level":3,"time":"2024-04-21T07:49:39+00:00","remoteAddr":"104.166.80.239","user":"--","app":"core","method":"GET","url":"/","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (X11; Linux i686; rv:109.0) Gecko/20100101 Firefox/120.0","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\DBAL\\Connection","typ
2024/04/21 08:07:57 [error] 387#387: *473 FastCGI sent in stderr: "PHP message: {"reqId":"iaAQ1PqYg8RUI6rtUFHD","level":3,"time":"2024-04-21T08:07:57+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 08:49:23 [error] 387#387: *475 FastCGI sent in stderr: "PHP message: {"reqId":"McVHBJwa2xlF2YqYlJnQ","level":3,"time":"2024-04-21T08:49:23+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 09:17:56 [error] 387#387: *477 FastCGI sent in stderr: "PHP message: {"reqId":"7siwrq4QKDUB7V4gBfN8","level":3,"time":"2024-04-21T09:17:56+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 10:19:00 [error] 387#387: *479 FastCGI sent in stderr: "PHP message: {"reqId":"e6cMgphzOAa5GqUd22cg","level":3,"time":"2024-04-21T10:19:00+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 10:26:11 [error] 387#387: *481 FastCGI sent in stderr: "PHP message: {"reqId":"m8XK5a7QxDX3LXKSofFw","level":3,"time":"2024-04-21T10:26:11+00:00","remoteAddr":"94.102.49.123","user":"--","app":"index","method":"GET","url":"/","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/45.0.2454.85 Safari/537.36","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"...; PHP message: {"reqId":"m8XK5a7QxDX3LXKSofFw","level":3,"time":"2024-04-21T10:26:11+00:00","remoteAddr":"94.102.49.123","user":"--","app":"core","method":"GET","url":"/","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/45.0.2454.85 Safari/537.36","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","c
2024/04/21 10:26:16 [error] 387#387: *483 FastCGI sent in stderr: "PHP message: {"reqId":"IJdsqsfiegS6eKLiqjTp","level":3,"time":"2024-04-21T10:26:16+00:00","remoteAddr":"94.102.63.27","user":"--","app":"index","method":"GET","url":"/","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/45.0.2454.85 Safari/537.36","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"e...; PHP message: {"reqId":"IJdsqsfiegS6eKLiqjTp","level":3,"time":"2024-04-21T10:26:16+00:00","remoteAddr":"94.102.63.27","user":"--","app":"core","method":"GET","url":"/","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/45.0.2454.85 Safari/537.36","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","cl
2024/04/21 10:26:28 [error] 387#387: *485 FastCGI sent in stderr: "PHP message: {"reqId":"X94nN8nm1XYuR2x5A8OW","level":3,"time":"2024-04-21T10:26:28+00:00","remoteAddr":"195.201.198.102","user":"--","app":"index","method":"GET","url":"/favicon.png","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"axios/0.24.0","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/p...; PHP message: {"reqId":"X94nN8nm1XYuR2x5A8OW","level":3,"time":"2024-04-21T10:26:28+00:00","remoteAddr":"195.201.198.102","user":"--","app":"core","method":"GET","url":"/favicon.png","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"axios/0.24.0","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/d
2024/04/21 10:26:28 [error] 387#387: *487 FastCGI sent in stderr: "PHP message: {"reqId":"D1pLSwh8CnNF54R0qxCJ","level":3,"time":"2024-04-21T10:26:28+00:00","remoteAddr":"195.201.198.102","user":"--","app":"index","method":"GET","url":"/favicon.ico","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"axios/0.24.0","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/p...; PHP message: {"reqId":"D1pLSwh8CnNF54R0qxCJ","level":3,"time":"2024-04-21T10:26:28+00:00","remoteAddr":"195.201.198.102","user":"--","app":"core","method":"GET","url":"/favicon.ico","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"axios/0.24.0","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/d
2024/04/21 10:39:25 [error] 387#387: *489 FastCGI sent in stderr: "PHP message: {"reqId":"ffBz3LKz3JWfx4JOkH0e","level":3,"time":"2024-04-21T10:39:25+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 11:17:57 [error] 387#387: *491 FastCGI sent in stderr: "PHP message: {"reqId":"UELah32I1QZ02YOUyIKN","level":3,"time":"2024-04-21T11:17:57+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 11:59:40 [error] 387#387: *493 FastCGI sent in stderr: "PHP message: {"reqId":"JcOIBqnBS8QuWMrLmKDs","level":3,"time":"2024-04-21T11:59:40+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 12:01:46 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"SLL5XRLzBimOTUonAG8x","level":3,"time":"2024-04-21T12:01:46+00:00","remoteAddr":"192.168.68.69","user":"--","app":"remote","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\...; PHP message: {"reqId":"SLL5XRLzBimOTUonAG8x","level":3,"time":"2024-04-21T12:01:46+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\DBAL\\Connection"
2024/04/21 12:01:46 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"ztu89GvyTPdoyeXn8VoA","level":3,"time":"2024-04-21T12:01:46+00:00","remoteAddr":"192.168.68.69","user":"--","app":"remote","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\...; PHP message: {"reqId":"ztu89GvyTPdoyeXn8VoA","level":3,"time":"2024-04-21T12:01:46+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\DBAL\\Connection"
2024/04/21 12:01:48 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"PumkkCDD1xJMWH5CUQIm","level":3,"time":"2024-04-21T12:01:48+00:00","remoteAddr":"192.168.68.69","user":"--","app":"remote","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/Photos","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doc...; PHP message: {"reqId":"PumkkCDD1xJMWH5CUQIm","level":3,"time":"2024-04-21T12:01:48+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/Photos","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\DBAL\\Conn
2024/04/21 12:01:48 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"s4IaSHFD6yaLT85jN0WB","level":3,"time":"2024-04-21T12:01:48+00:00","remoteAddr":"192.168.68.69","user":"--","app":"remote","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/Photos","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doc...; PHP message: {"reqId":"s4IaSHFD6yaLT85jN0WB","level":3,"time":"2024-04-21T12:01:48+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/Photos","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\DBAL\\Conn
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"BQXMFJ1unfJsMRKojhX0","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"remote","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/Photos/High%20Res%20Stuff","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"execute...; PHP message: {"reqId":"BQXMFJ1unfJsMRKojhX0","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/Photos/High%20Res%20Stuff","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"D
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"bIW897bBL1GeVz22YP1n","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"remote","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/Photos/High%20Res%20Stuff","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"execute...; PHP message: {"reqId":"bIW897bBL1GeVz22YP1n","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/Photos/High%20Res%20Stuff","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"D
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"OwqebiA5hZcINJddmd0Y","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/IMG_0599.JPG&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connect...; PHP message: {"reqId":"OwqebiA5hZcINJddmd0Y","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/IMG_0599.JPG&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"2su5y2ImH6IncPmgYxx0","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/Ph.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.ph...; PHP message: {"reqId":"2su5y2ImH6IncPmgYxx0","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/Ph.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"f
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"ZOvQlELkGEJsykv3Ms5f","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/tju29s0irqlb1.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Co...; PHP message: {"reqId":"ZOvQlELkGEJsykv3Ms5f","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/tju29s0irqlb1.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","l
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"1QmTaJplWvJXuSWckZv9","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/pokimane.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connect...; PHP message: {"reqId":"1QmTaJplWvJXuSWckZv9","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/pokimane.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"JiTFYeq6KKZkanZUgGpB","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/bGkt2x_xl.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connec...; PHP message: {"reqId":"JiTFYeq6KKZkanZUgGpB","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/bGkt2x_xl.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line"
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"c55BGCEbStys7Etso9Ai","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/bGk4w2.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connectio...; PHP message: {"reqId":"c55BGCEbStys7Etso9Ai","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/bGk4w2.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":26
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"I1r1sRNioOEVPp7sUZyv","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/RDT_20240418_0405362685342097460087546.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www...; PHP message: {"reqId":"I1r1sRNioOEVPp7sUZyv","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/RDT_20240418_0405362685342097460087546.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/priv
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"rvK0fyl8YmZvK2ZkHMhw","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/RDT_20240418_0402208487079481432511569.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www...; PHP message: {"reqId":"rvK0fyl8YmZvK2ZkHMhw","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/RDT_20240418_0402208487079481432511569.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/priv
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"eZ2uEBfRJ8IXKSSJr2re","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/mo1my8ivemzb1.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Co...; PHP message: {"reqId":"eZ2uEBfRJ8IXKSSJr2re","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/mo1my8ivemzb1.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","l
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"5xkpar1RqtQBIfac5TSe","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/poki.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection....; PHP message: {"reqId":"5xkpar1RqtQBIfac5TSe","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/poki.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"7q7Rd6UeqafRHeOibpKQ","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/uO2dgr.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connectio...; PHP message: {"reqId":"7q7Rd6UeqafRHeOibpKQ","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/uO2dgr.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":26
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"6R4OxNxjuMNrRy2tFzZ4","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/IMG_0599.JPG&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connect...; PHP message: {"reqId":"6R4OxNxjuMNrRy2tFzZ4","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/IMG_0599.JPG&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"P911Eq382rmWcJ47TxWb","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/Ph.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.ph...; PHP message: {"reqId":"P911Eq382rmWcJ47TxWb","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/Ph.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"f
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"PtMq0S19LPv6ofq3Mq4a","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/tju29s0irqlb1.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Co...; PHP message: {"reqId":"PtMq0S19LPv6ofq3Mq4a","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/tju29s0irqlb1.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","l
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"p0bUHBX6cpmjMAt3A75D","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/pokimane.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connect...; PHP message: {"reqId":"p0bUHBX6cpmjMAt3A75D","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/pokimane.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"Wr6r0q2wkMf4ntm3JabO","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/bGk4w2.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connectio...; PHP message: {"reqId":"Wr6r0q2wkMf4ntm3JabO","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/bGk4w2.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":26
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"XvJ9jGMObBmgUV854Hva","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/bGkt2x_xl.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connec...; PHP message: {"reqId":"XvJ9jGMObBmgUV854Hva","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/bGkt2x_xl.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line"
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"xJCGk8W99CyiMiLN6nrV","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/RDT_20240418_0405362685342097460087546.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www...; PHP message: {"reqId":"xJCGk8W99CyiMiLN6nrV","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/RDT_20240418_0405362685342097460087546.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/priv
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"m4BDjpyA0iuV5vWCG9EA","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/mo1my8ivemzb1.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Co...; PHP message: {"reqId":"m4BDjpyA0iuV5vWCG9EA","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/mo1my8ivemzb1.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","l
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"KO9cHiZqBOoaVyJJRgMN","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/RDT_20240418_0402208487079481432511569.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www...; PHP message: {"reqId":"KO9cHiZqBOoaVyJJRgMN","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/RDT_20240418_0402208487079481432511569.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/priv
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"nN2CO0eFnTC8IznG336C","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/poki.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection....; PHP message: {"reqId":"nN2CO0eFnTC8IznG336C","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/poki.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,
2024/04/21 12:01:53 [error] 388#388: *495 FastCGI sent in stderr: "PHP message: {"reqId":"Cjm1mPXITaoMWAHsyxCo","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"index","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/uO2dgr.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connectio...; PHP message: {"reqId":"Cjm1mPXITaoMWAHsyxCo","level":3,"time":"2024-04-21T12:01:53+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"GET","url":"/index.php/core/preview.png?file=Photos/High%20Res%20Stuff/uO2dgr.jpg&x=2048.0&y=2048.0&a=1&mode=cover","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":26
2024/04/21 12:02:29 [error] 388#388: *524 FastCGI sent in stderr: "PHP message: {"reqId":"scR6EdAVPpJqy5gQ1lsu","level":3,"time":"2024-04-21T12:02:29+00:00","remoteAddr":"192.168.68.69","user":"--","app":"remote","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\...; PHP message: {"reqId":"scR6EdAVPpJqy5gQ1lsu","level":3,"time":"2024-04-21T12:02:29+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\DBAL\\Connection"
2024/04/21 12:02:29 [error] 388#388: *524 FastCGI sent in stderr: "PHP message: {"reqId":"lT25F7gYTsLPSXhjIqQ3","level":3,"time":"2024-04-21T12:02:29+00:00","remoteAddr":"192.168.68.69","user":"--","app":"remote","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\...; PHP message: {"reqId":"lT25F7gYTsLPSXhjIqQ3","level":3,"time":"2024-04-21T12:02:29+00:00","remoteAddr":"192.168.68.69","user":"--","app":"core","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\DBAL\\Connection"
2024/04/21 12:05:13 [error] 388#388: *527 FastCGI sent in stderr: "PHP message: {"reqId":"B4VIpViFPOnXhWBs2SDs","level":3,"time":"2024-04-21T12:05:13+00:00","remoteAddr":"122.175.42.220","user":"--","app":"remote","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704","message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\...; PHP message: {"reqId":"B4VIpViFPOnXhWBs2SDs","level":3,"time":"2024-04-21T12:05:13+00:00","remoteAddr":"122.175.42.220","user":"--","app":"core","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704","message":"Exception thrown: Doctrine\\DBAL\\Exception","userAgent":"Mozilla/5.0 (iOS) Nextcloud-iOS/5.2.7","version":"28.0.4.1","exception":{"Exception":"Doctrine\\DBAL\\Exception","Message":"Failed to connect to the database: An exception occurred in the driver: SQLSTATE[HY000] [10] disk I/O error","Code":10,"Trace":[{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1654,"function":"connect","class":"OC\\DB\\Connection","type":"->"},{"file":"/app/www/public/3rdparty/doctrine/dbal/src/Connection.php","line":1081,"function":"getWrappedConnection","class":"Doctrine\\DBAL\\Connection","type":"->"},{"file":"/app/www/public/lib/private/DB/Connection.php","line":261,"function":"executeQuery","class":"Doctrine\\DBAL\\Connection
2024/04/21 12:17:59 [error] 388#388: *529 FastCGI sent in stderr: "PHP message: {"reqId":"3ETz12m2P2ufo9TCFWRT","level":3,"time":"2024-04-21T12:17:59+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 13:04:57 [error] 388#388: *531 FastCGI sent in stderr: "PHP message: {"reqId":"9UFOUwoubRWychzqh6kt","level":3,"time":"2024-04-21T13:04:57+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 13:20:46 [error] 388#388: *533 FastCGI sent in stderr: "PHP message: {"reqId":"F8IhWho4mGNIqekVZoz7","level":3,"time":"2024-04-21T13:20:46+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 13:59:29 [error] 388#388: *535 FastCGI sent in stderr: "PHP message: {"reqId":"dedxBuo2QDDiT0pXfScC","level":3,"time":"2024-04-21T13:59:29+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 14:18:58 [error] 388#388: *538 FastCGI sent in stderr: "PHP message: {"reqId":"OM8J6EpOinbDoLR7uIUg","level":3,"time":"2024-04-21T14:18:58+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
2024/04/21 14:58:02 [error] 388#388: *540 FastCGI sent in stderr: "PHP message: {"reqId":"UEcketXhnDht3xjxCePV","level":3,"time":"2024-04-21T14:58:02+00:00","remoteAddr":"122.175.42.220","user":"--","app":"PHP","method":"PROPFIND","url":"/remote.php/dav/files/Skylark9704/","message":"Error: Class \"OCA\\DAV\\Connector\\Sabre\\ExceptionLoggerPlugin\" not found at /app/www/public/remote.php#62","userAgent":"Mozilla/5.0 (Android) Nextcloud-android/3.28.2","version":"28.0.4.1","data":{"app":"PHP"}}" while reading response header from upstream, client: 122.175.42.220, server: _, request: "PROPFIND /remote.php/dav/files/Skylark9704/ HTTP/1.0", upstream: "fastcgi://127.0.0.1:9000", host: "nextcloud.skylark.cloud"
